### PR TITLE
Affix in ie10

### PR DESCRIPTION
**Pull request by MaaikeB**
PR - Affix in IE10 - scrollHeight #1073

### CODE REVIEW

**Comment by taion** [comment]
I wonder if we'd want to put this in `domUtils`.

Can you squash your commits?

**Comment by mtscout6** [comment]
I agree with @taion this seems reasonable to live in [`domUtils`](https://github.com/react-bootstrap/react-bootstrap/blob/master/src/utils/domUtils.js)

I'm not sure about the breaking build looks like some eslint problems @AlexKVal are those problems fixed on master? I believe that rebasing on the latest master should resolve that.

**Comment by mtscout6** [comment]
@jquense Are all the dom utils in flight with #1172 does this change make sense to be in `domUtils`?

**Comment by jquense** [comment]
They are in flight, but not all of them are going, just the ones that are covered by `dom-helpers` which this isn't. I would move it to domUtils, I'd also make sure it uses the owner document of the affix node, not just the global document, in case of iframes

**Comment by AlexKVal** [comment]
Eslint warnings are fixed now by https://github.com/react-bootstrap/react-bootstrap/pull/1185
I've restarted both CI-es.

**Comment by taion** [comment]
@MaaikeB 

Per this discussions here, go ahead and make this something like `function getDocumentHeight(document)` in `domUtils`, then squash your commits, and we should be good to go.

File: src/utils/domUtils.js [hunk 1]
```diff
@@ -79,6 +79,15 @@ function getOffset(DOMNode) {
 }
 
 /**
+ * Get the height of the document
+ *
+ * @returns {documentHeight: number}
+ */
+function getDocumentHeight() {
```
Comment (taion) [inline src/utils/domUtils.js]: You need to actually export this.

**Comment by taion** [comment]
Great - can you squash all of these down into 1 commit? LGTM otherwise. Need one more reviewer from @react-bootstrap/collaborators .

**Comment by mfunkie** [comment]
LGTM!
I'm going to have to start importing domUtils in my own projects, as I've hit this kinda stuff before.

**Comment by taion** [comment]
I don't know how `af6654f` ended up in there. Rather than go back again, I squashed and merged this PR manually.

**Pull request closed without merging**